tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/S3StorageSpec.scala

```diff
@@ -145,4 +148,24 @@ class S3StorageSpec extends StorageSpec {
       }
     }
   }
+
+  s"Registering an S3 file in-place" should {
+    "succeed" in {
+      val id      = genId()
+      val path    = s"$id/nexus-logo.png"
+      uploadLogoFile(path)
+      val payload = Json.obj("path" -> Json.fromString(path))
+      deltaClient.put[Json](s"/files/$projectRef/register/$id?storage=nxv:$storageId", payload, Coyote) {
+        (_, response) => response.status shouldEqual StatusCodes.Created
+      } >>
+        deltaClient.get[Json](s"/files/$projectRef/$id", Coyote) { (json, response) =>
+          response.status shouldEqual StatusCodes.OK
+          json.findAllByKey("_mediaType").headOption shouldEqual Some("image/png".asJson)
```

Comment:
There's an example in `ResourceMatchers`:
`json should haveField("_mediaType", "image/png")`